### PR TITLE
flow alerts

### PR DESCRIPTION
**Pull request by lessly-zuniga**
# Description

-flow alerts

Platform
- [x] iOS
- [ ] Android

Device
- [ ] Real device
- [x] Emulator

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/screens/app/payment/PaymentForm.tsx [hunk 1]
```diff
@@ -97,44 +132,84 @@ const PaymentForm: React.FC = () => {
...
+                  showSuccessPaymentEditAlert();
+                }
               },
               type: 'primary',
             },
             { label: t('global:cancel'), onPress: alert.hide, type: 'secondary' },
           ],
         });
         break;
-      case 'PaymentEdit':
```
Comment (EdgardoV) [inline src/screens/app/payment/PaymentForm.tsx:130]: Why did you delete this? @lessly-zuniga

**Review by EdgardoV — APPROVED** [review]

**Review by EmmanuelSalazarID — APPROVED** [review]
